lib/src/chart/bar_chart/bar_chart_data.dart

```diff
@@ -529,12 +537,14 @@ class BarChartRodStackItem with EquatableMixin {
         lerpDouble(a.fromY, b.fromY, t)!,
         lerpDouble(a.toY, b.toY, t)!,
         Color.lerp(a.color, b.color, t)!,
-        BorderSide.lerp(a.borderSide, b.borderSide, t),
+        label: a.label,
```

Comment:
Ahh, it's wrong.
It should be `b.label`. Otherwise it has a delay to update it

https://github.com/user-attachments/assets/18416784-cd21-441d-b836-770f02a476cd